app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def destroy
+    @users = User.all
+    @users.find(params[:id]).destroy
```

Comment:
why is this so difficult? 

Just User.find(...).destroy instead of two lines

Reply:
@okohuch Because we need that `@users` variable in response for json format. You'll see that if you look at the 10-11 lines.

Reply:
then you have repeated code

Reply:
in the index action you have the same @users = User.all

Reply:
please use a filter for setting `users` variable and rewrite this method into one line `User.find(...).destroy`